src/components/Main/Main.css

```diff
@@ -0,0 +1,20 @@
+.zodiac-main {
+  background-image: url('https://alchemy-react-zodiac.netlify.app/static/media/background.a66dac28.png');
+  display: flex;
+  flex-direction: column;
```

Comment:
if you change the flex-direction from column to row, the cards will wrap left to right.